src/code/qualification/runQualificationWorkflow.m

```diff
@@ -246,8 +240,13 @@ function runQualificationWorkflow(WSettings, ConfigurationPlan, TaskList, Observ
     if strcmp(TaskList{TaskListIndex}, 'PKRatioPlots')
         
         for PKRatioIndex=1:length(ConfigurationPlan.Plots.PKRatioPlots)
-            
-            PKRatioPlots=ConfigurationPlan.Plots.PKRatioPlots(PKRatioIndex);
+            % HideArtifacts optional field may change the size of PK Ratios
+            % leading to switch from struct to cell

```

Comment:
Don't understand this comment. How is the new _~~Hide~~Artifacts_ node supposed to change the structure of PKRatios? (PKRatios are even not on the same level as _Artifacts_ in the json file)

Reply:
My understanding is that Artifacts is an optional field of PKRatios.
Because it is optional, if Artifacts is only defined for a few PKRatios, Matlab output the PKRatios as cells. 
This issue happened for GOFMerged some time ago.